description: remove references to missing third_party/khronos_glcts dir
author: Michael Gilbert <mgilbert@debian.org>

--- a/build/all.gyp
+++ b/build/all.gyp
@@ -595,7 +595,6 @@
             '../chrome/chrome.gyp:performance_browser_tests',
             '../content/content_shell_and_tests.gyp:content_browsertests',
             '../gpu/gles2_conform_support/gles2_conform_test.gyp:gles2_conform_test',
-            '../gpu/khronos_glcts_support/khronos_glcts_test.gyp:khronos_glcts_test',
             '../gpu/gpu.gyp:gl_tests',
             '../gpu/gpu.gyp:angle_unittests',
             '../gpu/gpu.gyp:gpu_unittests',
@@ -622,7 +621,6 @@
             '../chrome/chrome.gyp:chrome',
             '../content/content_shell_and_tests.gyp:content_browsertests',
             '../gpu/gles2_conform_support/gles2_conform_test.gyp:gles2_conform_test',
-            '../gpu/khronos_glcts_support/khronos_glcts_test.gyp:khronos_glcts_test',
             '../gpu/gpu.gyp:gl_tests',
             '../gpu/gpu.gyp:angle_unittests',
             '../gpu/gpu.gyp:gpu_unittests',
--- a/build/gn_migration.gypi
+++ b/build/gn_migration.gypi
@@ -383,7 +383,6 @@
             '../extensions/extensions_tests.gyp:extensions_unittests',
             '../gin/gin.gyp:gin_shell',
             '../gpu/gles2_conform_support/gles2_conform_test.gyp:gles2_conform_test',  # TODO(GYP) crbug.com/471920
-            '../gpu/khronos_glcts_support/khronos_glcts_test.gyp:khronos_glcts_test',  # TODO(GYP) crbug.com/471903 to make this complete.
             '../ipc/ipc.gyp:ipc_perftests',
             '../mojo/mojo_base.gyp:mojo_common_unittests',
             '../ppapi/tools/ppapi_tools.gyp:pepper_hash_for_uma',
